### PR TITLE
Follow up to PR 12160 to update the DB version of the templates

### PR DESCRIPTION
**Pull request by aydun**
This just updates the DB version of the event registration templates following the changes in PR 12160

### CODE REVIEW

**Comment by aydun** [comment]
@eileenmcnaughton - here's the follow-up to https://github.com/civicrm/civicrm-core/pull/12160  Is '5.4.alpha2' the right version to use?

**Comment by eileenmcnaughton** [comment]
@aydun so we already have this merged to 5.4 & I believe your template update is merged already. 

```
  /**
   * Get any templates that have been updated.
   *
   * @return array
   */
  protected function getTemplateUpdates() {
    return [
      [
        'version' => '5.4.alpha1',
        'upgrade_descriptor' => ts('Use email greeting at top where available'),
        'templates' => [
          ['name' => 'membership_online_receipt', 'type' => 'text'],
          ['name' => 'membership_online_receipt', 'type' => 'html'],
          ['name' => 'contribution_online_receipt', 'type' => 'text'],
          ['name' => 'contribution_online_receipt', 'type' => 'html'],
          ['name' => 'event_online_receipt', 'type' => 'text'],
          ['name' => 'event_online_receipt', 'type' => 'html'],
        ]
      ],
    ];
  }
```

So I think your text & html will get picked up already - a weakness is that the upgrade_descriptor only mentions one descriptor whereas we now have multiple :-(

If subject also changed then I think a PR to the 5.4 branch altering the above to include
```
['name' => 'event_online_receipt', 'type' => 'subject'],
```

Will work

File: CRM/Upgrade/Incremental/MessageTemplates.php [hunk 1]
```diff
@@ -81,6 +81,15 @@ protected function getTemplateUpdates() {
           ['name' => 'event_online_receipt', 'type' => 'html'],
         ]
       ],
+      [
+        'version' => '5.4.alpha2',
```
Comment (seamuslee001) [inline CRM/Upgrade/Incremental/MessageTemplates.php]: i think this should be 5.5.alpha1 i think

Comment (eileenmcnaughton) [reply CRM/Upgrade/Incremental/MessageTemplates.php]: yep - if it is against master - see above for why I think 5.4 rc makes sense

**Comment by aydun** [comment]
Ok... is this what you mean?

**Comment by eileenmcnaughton** [comment]
@aydun - yes - but rebase onto the 5.4 branch since the rc has been cut & we should include this with the other lines

**Comment by seamuslee001** [comment]
@aydun Just needed to change the basis of the PR :-) @eileenmcnaughton you ok with this being merged to RC then?

**Comment by eileenmcnaughton** [comment]
@seamuslee001 yes - added merge on pass - in theory adding these templates to the array like this is 'always safe'

**Comment by seamuslee001** [comment]
Merging as per the tag

**Comment by aydun** [comment]
@seamuslee001 Thanks for doing that - would you mind sharing the git commands you needed to do for this?  I did a 'git rebase 5.4' but clearly there was something else needed.

**Comment by eileenmcnaughton** [comment]
@aydun this is what I do

1 ) ``git fetch -f origin ``
(to make sure I have the latest locally
2) `` git rebase -i origin/master`` (or `` git rebase -i origin/5.4`` in this case
3) `` git push -f myrepo/mybranch``

**Comment by seamuslee001** [comment]
@eileenmcnaughton that is what @aydun had done but then all i did was just edit the PR to change the branch of the civicrm-core repo the PR was targeting @aydun

**Comment by eileenmcnaughton** [comment]
@seamuslee001 ah

**Pull request merged**